package/rebuild-todo

```diff
@@ -219,6 +224,22 @@ read <&1
 
 pkgctl repo clone "${packages[@]}"
 
+if ((IMPORT_KEYS)); then
+	echo -e "Importing PGP keys...\n"
+	# Only add paths that actually have key(s) to import, ignore errors and don't exit on paths that don't
```

Comment:
I think this comment is not complete

Reply:
It is, but the phrasing can be improved I guess. I'll try to come up with something clearer

Reply:
I think if you just switch the order it already makes more sense:
```suggestion
	# Only add paths that actually have key(s) to import and ignore paths that don't, don't exit on error 
```